webapp/src/contexts/ConferenceContext/methods/connect.ts

```diff
@@ -0,0 +1,90 @@
+import { ClientCallType, createCallSignals, createInfinityClient, createInfinityClientSignals } from '@pexip/infinity'
+import { ConferenceActionType, type ConferenceAction } from '../ConferenceAction'
+
+interface ConnectParams {
+  host: string
+  conferenceAlias: string
+  hostPin: string
```

Comment:
Right now it's only available with HostPin. SSO and guest pin can be used if we configure the Web App 3 pop up instead of the embedded experience. The plan is to also integrate an External Policy which will give us more control.